src/panels/lovelace/sections/hui-section.ts

```diff
@@ -131,6 +124,14 @@ export class HuiSection extends ReactiveElement {
       }
       if (changedProperties.has("lovelace")) {
         this._layoutElement.lovelace = this.lovelace;
+        this._cards.forEach((element) => {
+          try {
+            element.hass = this.hass;
```

Comment:
Do we need to set hass again? We already did that on line 118?

Reply:
I don't think so